controllers/src/test/scala/cromwell/pipeline/controller/ProjectFileControllerTest.scala

```diff
@@ -51,7 +51,7 @@ class ProjectFileControllerTest extends AsyncWordSpec with Matchers with Scalate
     }
 
     "upload file" should {
-      val projectFile = ProjectFile(Paths.get("folder/test.txt"), projectFileContent)
+      val projectFile = ProjectFile(path, projectFileContent)
```

Comment:
That little guy was just too annoying? :)

Reply:
Yep. And we need to cimbine all vriables into one place.